ui/lib/apps/SlowQuery/pages/Detail/index.tsx

```diff
@@ -143,26 +149,58 @@ function DetailPage() {
                       </Descriptions.Item>
                     )
                 })()}
-                <Descriptions.Item
-                  span={2}
-                  multiline={detailExpand.plan}
-                  label={
-                    <Space size="middle">
-                      <TextWithInfo.TransKey transKey="slow_query.detail.head.plan" />
-                      <Expand.Link
-                        expanded={detailExpand.plan}
-                        onClick={togglePlan}
-                      />
-                      <CopyLink data={data.plan ?? ''} />
-                    </Space>
-                  }
-                >
-                  <Expand expanded={detailExpand.plan}>
-                    <Pre noWrap>{data.plan}</Pre>
-                  </Expand>
-                </Descriptions.Item>
-              </Descriptions>
+                {(() => {
+                  if (data.plan)
```

Comment:
What's different between lambda(`() => if (x) return (<Component />)`) and logical operator(`x && (<Component />)`) in react? It's looks like lighter to use lambda function.